docs/API.md

```diff
@@ -613,10 +702,19 @@ __Parameters__
 | Param  |  Type | Description  |
 |---|---|---|
 | `bucketName`  | _string_  | Name of the bucket.  |
-|`objectName`   |_string_   | Name of the object.  |
+| `objectName`   |_string_   | Name of the object.  |
 | `filePath`  | _string_  | Path of the file to be uploaded.  |
 | `metaData`  | _Javascript Object_  | Metadata of the object.  |
-| `callback(err, etag)`  |  _function_ | Non-null `err` indicates error, `etag` _string_ is the etag of the object uploaded. If no callback is passed, a `Promise` is returned. |
+| `callback(err, objInfo)` _function_: non null `err` indicates error, `objInfo` _object_ is the information about the object uploaded which contains `versionId` string and `etag` string.
+
+__Return Value__
+
+| Value  |  Type | Description  |
+|---|---|---|
+| `err` | _object_ | Error in case of any failures
+| `objInfo.etag`  | _string_  | `etag` of an object  |
+| `objInfo.versionId`  | _string_  | `versionId` of an object (optional)  |
+
 
 __Example__
 
```

Comment:
Example code snippet needs to be updated

Reply:
I have updated the snippet to reflect the same.